test/index.html

```diff
@@ -1,97 +0,0 @@
-<!DOCTYPE HTML>
```

Comment:
Yep, just have separate watch/build tasks that are specifically for tests and then include that bundled output. [Example](https://github.com/brightcove/videojs-playlist/blob/master/package.json#L12-L16).